iocs/PICamIOC/iocBoot/iocPICam/st.cmd

```diff
@@ -2,7 +2,7 @@ errlogInit(20000)
 
 < envPaths
 #epicsThreadSleep(20)
-dbLoadDatabase("$(TOP)/dbd/PICamApp.dbd")
+dbLoadDatabase("/home/towa/EPICS/support/areaDetector/ADPICam/iocs/PICamIOC/dbd/PICamApp.dbd")
```

Comment:
The original line using $(TOP) is correct.  Running "make" in the iocPICam directory will produce an envPaths file that correctly defines TOP.